secator/tasks/getasn.py

```
@@ -0,0 +1,61 @@
+import os
```

Flake8 (7.3.0)

[error] 'os' imported but unused

(F401)